src/models/subject.ts

```diff
@@ -0,0 +1,50 @@
+import { Classification } from "./decisions/classification";
+import { Verification, VerificationSubject } from "./decisions/verification";
+
+export interface Tag {
+    id?: number;
+    text: string;
+}
+
+export interface ISubjectWrapper {
+    subject: VerificationSubject;
+    identifier: symbol;
+
+    url: string;
+    decisions: [Verification, ...Classification[]] | Classification[];
+    tag: string;
+}
+
+// the Verification model is emitted by the oe-verification-grid as a CustomEvent
+export class SubjectWrapper implements ISubjectWrapper {
+    public constructor(data: ISubjectWrapper) {
+        this.subject = data.subject;
+        this.decisions = data.decisions ?? [];
+        this.url = data.url;
+        this.tag = data.tag;
+
+        this.identifier = Symbol("decision");
+    }
+
+    // aka: context
+    // this is the native data model used by the host application
+    // or this could be the csv row
+    public identifier: symbol;
+    public subject: VerificationSubject;
+
+    // each row can only have one verification decision
+    // but can have multiple classification decisions
+    // verification decisions will be reflected in the oe-confirmed
+    // column, while each classification will get its own row
+    public decisions: [Verification, ...Classification[]] | Classification[];
```

Comment:
based on what else i've seen in this review, this array should be a getter and you should store Verifications and classifications separately (or as a map)